newclient/src/modules/txs/handlers/TxDetails/TxDetails.vue

```diff
@@ -260,14 +260,14 @@ export default class TxDetails extends Mixins(NumberFormatMixin) {
      *
      * @return {String}
      */
-    get txFee(): string | null {
+    get txFee(): any {
```

Comment:
can you change this to:  'FormattedNumber' ?

Reply:
changed!